Xamarin.Essentials/WebAuthenticator/WebAuthenticator.uwp.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Linq;
+using System.Xml.XPath;
+
+namespace Xamarin.Essentials
+{
+    public static partial class WebAuthenticator
+    {
+        static Task<AuthResult> PlatformAuthenticateAsync(Uri url, Uri callbackUrl)
+        {
+            redirectUri = callbackUrl;
+
+            tcsResponse = new TaskCompletionSource<AuthResult>();
+
+            if (!IsUriProtocolDeclared(callbackUrl.Scheme))
+                throw new InvalidOperationException($"You need to declare the windows.protocol usage of the protocol/scheme `{callbackUrl.Scheme}` in your AppxManifest.xml file");
+
+            Windows.System.Launcher.LaunchUriAsync(url).AsTask();
```

Comment:
Is there a reason for the web browser here instead of the Authentication Broker? It does mostly the same thing I _think_.

Reply:
uhh yep, in my mind i had done this already ;)